dask_deltatable/core.py

```diff
@@ -161,16 +160,15 @@ def read_delta_table(self, **kwargs) -> dd.core.DataFrame:
         pq_files = self.get_pq_files()
         if len(pq_files) == 0:
             raise RuntimeError("No Parquet files are available")
-        parts = [
-            delayed(
-                self.read_delta_dataset,
-                name="read-delta-table-" + tokenize(self.fs_token, f, **kwargs),
-            )(f, **kwargs)
-            for f in list(pq_files)
-        ]
-        meta = self._make_meta_from_schema()
-        verify_meta = kwargs.get("verify_meta", False)
-        return from_delayed(parts, meta=meta, verify_meta=verify_meta)
+
+        label = "read-delta-table-"
+        return dd.from_map(
+            partial(self.read_delta_dataset, **kwargs),
+            pq_files,
+            meta=self.meta,
+            label=label,
+            token=f"{label}{tokenize(self.fs_token, **kwargs)}",
+        )
```

Comment:
```suggestion
        return dd.from_map(
            partial(self.read_delta_dataset, **kwargs),
            pq_files,
            meta=self.meta,
            label="read-delta-table",
            token=tokenize(self.fs_token, **kwargs),
        )
```
I *think* this is what you want, since `from_map` should be using `label=` and `token=` to produce a Layer name like [`f"{label}-{token}"`](https://github.com/dask/dask/blob/499f4055da707fa76d06a2b79f408f124eee4723/dask/dataframe/io/io.py#L1014)

Reply:
Nice -- that's cleaner too : )